app/views/providers/means_reports/_uploaded_bank_statements.html.erb

```diff
@@ -4,33 +4,24 @@
 
 <%= govuk_summary_card(title: t(".bank_statement_heading"),
                        heading_level: 3,
-                       html_attributes: { id: "app-check-your-answers__bank_statements_client__card" }) do |card| %>
+                       html_attributes: { id: "app-check-your-answers__bank_statements_client__card" }) do |card|
+      card.with_summary_list(actions: false, html_attributes: { id: "app-check-your-answers__bank_statements_client__summary" }) do |summary_list|
+        if @legal_aid_application.client_uploading_bank_statements?
+          summary_list.with_row(html_attributes: { id: "app-check-your-answers__bank_statements_client_summary" }) do |row|
+            row.with_key(text: has_partner ? t(".bank_statement_question_with_partner") : t(".bank_statement_question"), classes: "govuk-!-width-one-half")
```

Comment:
I think this should be `govuk-!-width-one-third` so that it's consistent with the CYA pages.